### PR TITLE
complete task 5

### PR DESCRIPTION
**Pull request by AmadReihun**
completed task 5

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -126,4 +126,47 @@ describe("app", () => {
...
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article.article_img_url).toBe("string");
+            expect(typeof article.count).toBe("string");
+            expect(typeof article.body).toBe("undefined");
+          });
+
+          // Checking that the articles are sorted by date in descending order
+
+          for (let i = 0; i < articles.length-1; i++) {
+          expect(articles[i+1].created_at <= articles[i].created_at).toBe(true)
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:161]: Please use jest-sorted here.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -126,4 +126,47 @@ describe("app", () => {
...
+          }
+        });
+    });
+
+    test("to get error 404 if the path is wrong", () => {
+      return request(app)
+        .get("/api/wrongpath")
+        .expect(404)
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:169]: We need to check what message we are getting as well.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -126,4 +126,47 @@ describe("app", () => {
...
+          
+          articles.forEach((article) => {
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.title).toBe("string");
+            expect(typeof article.article_id).toBe("number");
+            expect(typeof article.topic).toBe("string");
+            expect(typeof article.created_at).toBe("string");
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article.article_img_url).toBe("string");
+            expect(typeof article.count).toBe("string");
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:154]: The property should be called comment_count.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Ahmad, just a few things to refactor. Please see my comments.

File: __tests__/app.test.js (hunk 1 above)
Comment (AmadReihun) [reply __tests__/app.test.js:161]: Thanks for the hint, done

File: __tests__/app.test.js (hunk 2 above)
Comment (AmadReihun) [reply __tests__/app.test.js:169]: Hi there,
The wrong path message is generated by in-built error detection and the response body is an empty object.
Is there any way to address this please?
Cheers

File: __tests__/app.test.js (hunk 3 above)
Comment (AmadReihun) [reply __tests__/app.test.js:154]: Added AS comment_count on db query and sorted, thanks

**Pull request merged**